# request fields
geometry_msgs/PoseStamped goal
---
# response fields
nav_msgs/Path global_plan